docs/data-sources/cluster.md

```diff
@@ -43,6 +43,7 @@ data "ybm_cluster" "example_cluster"{
 - `cluster_type` (String) The type of the cluster.
 - `cluster_version` (String)
 - `credentials` (Attributes) (see [below for nested schema](#nestedatt--credentials))
+- `database_version` (String) The version of the database. Stable or Preview.
```

Comment:
I would name it database_track because Database version is of format 2.12.5 and has a different meaning.

Reply:
Sure, I will rename it.